.github/workflows/dist.yml

```diff
@@ -0,0 +1,20 @@
+name: dist
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "stable"
```

Comment:
shouldn't this be the same version used elsewhere?

Reply:
I noticed this when copying the action from yggdrasil's workflow. We could probably drop this `with` option entirely; it's not relevant for the purposes of making a tarball.